test/functionalTest/cases/0951_get_v2_param_type/get_v2_param_type.test

```diff
@@ -181,12 +181,12 @@ Date: REGEX(.*)
 06. GET /v2/entities/E1/attrs/A1/value?type=T1
 ==============================================
 HTTP/1.1 200 OK
-Content-Length: 11
+Content-Length: 13
 Content-Type: text/plain
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
-V1 A1 E1/T1
+"V1 A1 E1/T1"
```

Comment:
This is another fix different than the one already described in the CNR about Accept header processing. I sugges to describe it in an additional entry in CNR (in this case, without associated issue number, as far as I remember).